spec/requests/users_controller_spec.rb

```diff
@@ -4684,7 +4684,7 @@ def create_and_like_post(likee, liker)
       it "should redirect to login page for anonymous user when profiles are hidden" do
         SiteSetting.hide_user_profiles_from_public = true
         get "/u/#{user.username}/card.json"
-        expect(response).to redirect_to "/login"
+        expect(response).to have_http_status(:forbidden)
```

Comment:
we should update the name of the test here as well, it should no longer read "should redirect to..."